styles/chatgpt/catppuccin.user.css

```diff
@@ -931,6 +931,10 @@
       --tw-ring-offset-color: @base;
     }
 
+    [style*="color: rgb(204, 0, 0)"] {
```

Comment:
Can we add a comment here to help us remember what this styles? From your screenshot in the original issue it looked like LaTeX math expressions or something?